bundles/org.openhab.binding.opensprinkler/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -109,17 +167,51 @@
 		<category>Switch</category>
 	</channel-type>
 
+	<channel-type id="signalStrength" advanced="true">
```

Comment:
There is the system Channel for the signal strength. See https://www.openhab.org/docs/developer/bindings/thing-xml.html#system-state-channel-types

Reply:
So it is preferred to convert the dB scale with a defined dimension, into a meaningless/non dimensional value of 0 to 4 value? Seems strange to leave a dimension and start guessing where to convert a db values into a made up value of say 1?

Reply:
OpenHAB's job is to abstract things. This abstraction gives a consistent view to all Things. And let's be honest, you don't need a resolution of 1db... But this is not the right place to discuss that.

Reply:
Done: Changed it over to system channel as suggested.